flang/test/Driver/fcc_override.f90

```diff
@@ -0,0 +1,12 @@
+! RUN: env FCC_OVERRIDE_OPTIONS="#+-Os +-Oz +-O +-O3 +-Oignore +a +b +c xb Xa Omagic ^-###  " %flang -target x86_64-unknown-linux-gnu %s -O2 b -O3 2>&1 | FileCheck %s
```

Comment:
Does `+`, `x`,`X` have special meanings? Is that documented anywhere?

I think we should document `FCC_OVERRIDE_OPTIONS`.  One possible location is https://github.com/llvm/llvm-project/blob/main/flang/docs/FlangDriver.md

Reply:
`--target=x86_64-unknown-linux-gnu` instead of the long deprecated `-target `